apps/dotcms-ui/src/app/api/services/dot-crud/dot-crud.service.ts

```diff
@@ -20,7 +20,7 @@ export class DotCrudService {
      * @returns Observable<any>
      * @memberof CrudService
      */
-    public postData(baseUrl: string, data: any): Observable<any> {
+    public postData(baseUrl: string, data: unknown): Observable<unknown> {
```

Comment:
```suggestion
    public postData<T>(baseUrl: string, data: T): Observable<unknown> {
```

Should we use a [generic](https://www.typescriptlang.org/docs/handbook/2/generics.html) here?